test/integration/test-cases/regression/input/issue.1682.yaml

```
@@ -0,0 +1,17 @@
+repositories:
+- name: bitnami
+  url: https://charts.bitnami.com/bitnami
+
+releases:
+  - name: elasticsearch
+    chart: bitnami/elasticsearch
+    version: 19.0.1
+    jsonPatches:
+    - target:
+        version: v1
+        kind: StatefulSet
+        name: elasticsearch-master
+      patch:
+      - op: add
+        path: /spec/template/spec/containers/0/volumeMounts/0/subPathExpr
+        value: elasticsearch/$(MY_POD_NAME)
```

Shouldn't it be two releases? Like below:
```suggestion
releases:
  - name: elasticsearch18
    chart: bitnami/elasticsearch
    version: 18.0.0
  - name: elasticsearch19
    chart: bitnami/elasticsearch
    version: 19.0.0
```

---

@kevinleturc oh. you are right. I pasted the error code.